### PR TITLE
Add generic questions for metadata and refactor Question class

### PR DESCRIPTION
**Pull request by JinTotonic**

### CODE REVIEW

File: src/AentHelper.php [hunk 1]
```diff
@@ -138,4 +138,51 @@ public function question(string $question): \TheAentMachine\Helper\Question
...
+     * @return Helper\ChoiceQuestion
+     */
+    public function choiceQuestion(string $question, array $choices): \TheAentMachine\Helper\ChoiceQuestion
+    {
+        return new \TheAentMachine\Helper\ChoiceQuestion($this->questionHelper, $this->input, $this->output, $question, $choices);
+    }
+
+    public function setEnvType(): string
+    {
+        $envType = $this->choiceQuestion('Select your environment type', ['DEV', 'TEST', 'PROD'])
```
Comment (gulien) [inline src/AentHelper.php]: We should replace those with some constants (will be easier to retrieve metadata).

File: src/Helper/ChoiceQuestion.php [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+namespace TheAentMachine\Helper;
+
+use Symfony\Component\Console\Helper\QuestionHelper;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+/**
+ * A helper class to easily create choice questions.
+ */
+class ChoiceQuestion extends BaseQuestion
```
Comment (gulien) [inline src/Helper/ChoiceQuestion.php]: We should use some kind of validator for help text and default value?

File: src/AentHelper.php (hunk 1 above)
Comment (JinTotonic) [reply src/AentHelper.php]: Oki doki !

File: src/Helper/ChoiceQuestion.php (hunk 2 above)
Comment (JinTotonic) [reply src/Helper/ChoiceQuestion.php]: Symfony's ChoiceQuestion has its own validator which is pretty good (you can multiselect using commas or spaces as separator)

**Pull request merged**